lib/dip/commands/runners/docker_compose_runner.rb

```diff
@@ -57,6 +68,17 @@ def published_ports
             []
           end
         end
+
+        def update_command_for_profiles
+          # NOTE: When using profiles, the method is always `up`.
+          #       This is because `docker-compose` does not support profiles
+          #       for other commands. Also, run options need to be removed
+          #       because they are not supported by `up`.
+          command[:compose][:method] = "up"
+          command[:command] = ""
+          command[:service] = nil
```

Comment:
I don't think we should nullify the service, because of Docker Compose supports starting specific services with profiles. https://docs.docker.com/compose/profiles/

Reply:
Yeah, you are right. Although this look a bit strange.
Example compose file (similar to compose docs)
```
version: "3.9"
services:
  web:
    image: nginx

  backend:
    image: memcached
    profiles: ["dev"]
    depends_on:
      - db

  db:
    image: redis
    profiles: ["dev"]

  runner:
    image: alpine
    command: ["tail", "-f", "/dev/null"]
    profiles: ["debug"]
    depends_on:
      - db
```
Command `docker compose --profile dev up web` will not start any services from `dev` profile.
But `docker compose --profile dev up runner` will start only `db`.
So, `service` nullification can be remove for sure. But profiles from the Dip config may or may not be started in this case. I'll update PR soon.